pom.xml

```diff
@@ -34,11 +34,11 @@
   </licenses>
 
   <modules>
-    <module>sdks/java/client-java-core</module>
-    <module>sdks/java/client-java-jersey</module>
-    <module>sdks/typescript/client-typescript-core</module>
-    <module>sdks/typescript/client-typescript-eventsource</module>
-    <module>sdks/dart/client-dart-api</module>
+<!--    <module>sdks/java/client-java-core</module>-->
```

Comment:
looks like there is a conflict in this branch with master

Reply:
resolved